aw_server/config.py

```diff
@@ -6,12 +6,14 @@
 port = "5600"
 storage = "peewee"
 cors_origins = ""
+custom_watcher_visualizations = "{}"
```

Comment:
Avoid JSON in the config. Use the TOML format.

Example:

```toml
[server]
...

[server.custom_visualizations]
aw-watcher-xxx = "/home/user/code/aw-watcher-xxx/static"
```

Maybe call it `custom_static` instead of `custom_visualizations` though? (in case it could generalize)

Reply:
Sounds good. I will do that.